release/src/release-notes.ts

```diff
@@ -79,19 +81,35 @@ enum IssueType {
   underTheHoodIssues = "underTheHoodIssues",
 }
 
+const otherProductCategory = "Other";
+
+type ProductCategory =
+  | "Administration"
+  | "Database"
+  | "SDK"
+  | "Embedding"
+  | "Operation"
+  | "Organization"
+  | "Querying"
+  | "Reporting"
+  | "Visualization"
+  | "Other";
+
 // Product area labels take the form of "Category/Subcategory", e.g., "Querying/MBQL"
 // We're only interested in the main product category, e.g., "Querying"
-enum ProductCategory {
-  administration = "Administration",
-  database = "Database",
-  embedding = "Embedding",
-  operation = "Operation",
-  organization = "Organization",
-  querying = "Querying",
-  reporting = "Reporting",
-  visualization = "Visualization",
-  other = "Other",
-}
+// The position in list matters, as it determines the order in which issue labels are tested against items in this list
+const productCategories: ProductCategory[] = [
+  "Administration",
+  "Database",
+  "SDK",
+  "Embedding",
```

Comment:
I guess this is so SDK takes precedence, and if it's not SDK it falls back to Embedding.

Reply:
Correct. I added a comment for the array that mentions order